dspace/config/modules/rest.cfg

```diff
@@ -54,143 +54,4 @@ rest.properties.exposed = google.recaptcha.mode
-rest.report-mime-document-image = image/jpeg,image/jp2
-
-# Minimum size for a supported PDF in the repository
-# PDF bitstreams smaller than this size will be highlighted in a report.
-# PDF files smaller than this size are potentially corrupt.
-rest.report-pdf-min-size = 20000
-
-# Maximum size for a typical PDF in the repository
-# PDF bitstreams larger than this size will be highlighted in a report.
-# PDF files larger than this size may be slow to retrieve.
-rest.report-pdf-max-size = 25000000
-
-# Minimum size for a thumbnail - could indicate a corrupted original
-# Thumbnail bitstreams smaller than this size will be highlighted in a report.
-# Thumbnail files smaller than this size are potentially corrupt.
-rest.report-thumbnail-min-size = 400
-
-# Bitstream descriptor to identify generated thumbnails
-# The ImageMagick Thumbnail generator tags the thumbnails it has created with a standard description.
-# This description identifies thumbnails that can safely be re-generated.
-rest.report-gen-thumbnail-desc = Generated Thumbnail
-
-#### Metadata Filtering by Regular Expression #####
-# Used by org.dspace.rest.filter.ItemFilterDefsMeta
-# This class filters items based on metadata properties.
-# These filters are useful for filtering a small set of items.  These filters will be inefficient as a query tool.
-
-# regex to detect compound subjects - detect subject terms that should be split into individual terms
-rest.report-regex-compound-subject = .*;.*
-
-# regex to detect compound authors - detect author/creator names taht should be split into individual fields
-rest.report-regex-compound-author = .* and .*
-
-# regex to detect unbreaking metadata - detect long unbreaking text that may not render properly on a page
-# Be sure to escape commas to prevent Apache Config from breaking regex into array values
-rest.report-regex-unbreaking = ^.*[^ ]{50\,50}.*$
-
-# regex to detect url in description - detect description fields that contain URL's
-rest.report-regex-url = ^.*(http://|https://|mailto:).*$
-
-# regex to identify full text content from the provenance field
-# This test has been used to identfiy "full-text" content when harvesting DSpace content by Summon
-rest.report-regex-fulltext = ^.*No\\. of bitstreams(.|\\r|\\n|\\r\\n)*\\.(PDF|pdf|DOC|doc|PPT|ppt|DOCX|docx|PPTX|pptx).*$
-
-# regex to identify very long metadata fields that may be slow to render
-# Be sure to escape commas to prevent Apache Config from breaking regex into array values
-rest.report-regex-long = ^[\\s\\S]{6000\,}$
-
-# regex to identify partial XML entities within a description field (a frequent problem found in ProQuest ETD's)
-rest.report-regex-xml-entity = ^.*&#.*$
-
-# regex to identify non ascii characters in metadata
-rest.report-regex-non-ascii = ^.*[^\\p{ASCII}].*$
-
-# The maximum number of results to return for 1 request
-rest.search.max.results = 100
+rest.properties.exposed = handle.canonical.prefix
```

Comment:
New line

Reply:
Thanks @rmathew1011 for the review.  To be honest, I don't think having a blank line at the end of these cfg files makes a difference :)  But I went ahead and added it back anyways.  So this is resolved.  (Note: I rebased this tiny change into the existing commit so there is no separate commit)